Library/Formula/openvpn.rb

```diff
@@ -25,12 +35,22 @@ def install
         "security/pam_appl.h", "pam/pam_appl.h"
     end
 
+    opts = []
+    if build.with? "pkcs11"
+        opts << "--enable-pkcs11"
+        ENV.append_path "PKG_CONFIG_PATH", "/usr/local/lib/pkgconfig"
```

Comment:
Please don't add the entire top-level `pkgconfig` path back in. We deliberately keep that out of builds because most things attempt to scoop things up from the environment they haven't actually been told to use under the pretence of being helpful.